addons/binding/org.openhab.binding.solarview/README.md

```diff
@@ -0,0 +1,234 @@
+# solarview Binding
+
+This binding is for users coming from the photovoltaics monitoring-system "Solarview". This measurement and monitoring
+environment provides access to a lot of different energy monitoring devices like inverters and meter readers.
+
+For details about the feature, see the following website ![Solarview](http://www.solarview.info) 
+
+The binding is fairly complete and supports the following monitoing functionalities.
+
+* energy,
+* power,
+* voltage and
+* temperature
+
+at different levels/devices.
+
+
+## Binding Configuration
+
+The binding can be configured in the file `services/solarview.cfg`.
+
+| Property | Default | Required | Description |
+|----------|---------|:--------:|-------------|
+| host     | localhost |   No    | Hostname or IP address of the solarview system |
+| refreshInterval  | 60 | No | Timeout in seconds |
+
+
+## Supported Things
+
+The Solarview system provides an umbrella view onto the energy flow of a complete domestic home.
+Therefore three main views exist:
+* the energy production (i.e. the photovolatic system),
+* the energy import (i.e. any demand not locally satisfied by the production system),
+* the energy injection (i.e. any demand not locally needed within the domestic home),
+
+In addition there are three more production-specific views onto the photovolatic system with
+
+* productionInverterOne,
+* productionInverterTwo and
+* productionInverterThree.
+
+## Discovery
+
+Unfortunatelly there is no way to discover the Solarview system in the local network. Beware that all Solarview monitoring sources have to be added to the local Solarview installation as described in the Solarview setup procedure.
+
+## Thing Configuration
+
+The Solarview Thing requires the hostname (or IP address) as a configuration value in order for the binding to know how to access it.
+Additionally, a refresh interval, used to poll the Solarview system, can be specified (in seconds).
+
+In the thing file, this looks e.g. like
+```
+Thing solarview:energyProduction:home [ hostname="my.sweet.home" ]
+Thing solarview:energyInjection:anotherBrick [ hostname="192.168.1.1", refreshInterval=60 ]
```

Comment:
I don't know the solarview system very well. However, it looks to me like it is one device running somewhere (RPi/FritzBox) in your network, so it definitely makes sense to have a "bridge" what wraps all the things together so you only have to configure the connection details once. If somebody has solarview running on two or more different servers, than this user simply would have to define several bridges, each with it's own network configuration and its relevant things underneath.

Regardin the `productionInverterOne...Three`: They look absolutely identical to me, so from a logical point of view I don't see a need for defining three separate thing _types_ for them. It's one and the same thing type which the user uses to define up to three things, just with a different configuration parameter (I get it right that these are really different inverters and not only strings within the same inverter?) By the way, on the website I just read that it supports up to 9 inverters - so this approach would even easily allow supporting an arbitrary number of inverters on openHAB side. The user would just have to provide the required configuration parameter to identify them (whatever the API needs, you know that better than me...)